### DISCUSSION TITLE
Any release planed?

### DISCUSSION THREAD

**Discussion by irieger** [bbc/bmx#101] [General]
Hey,

I was wondering if any release is planed? As there was quite a few things added and fixed since 1.2 I'd really appreciate to easily being able to use. As I'm using the conan recipe I created for the conan-center-index (https://conan.io/center/recipes/bmx) in a modified way, I have a state from a few months ago when I last touched it. However it would be nice to not have to fall back to manual modifications etc., but to have a tag and then quickly do a version bump for the conan recipe.

Any chance we could see a release in the near future?

Cheers and thanks for the work,
Ingmar

**Comment by philipnbbc** [comment]
Sorry for not replying earlier; for some reason GitHub didn't send a notification email for this message.

It's been a while so a release makes sense. I'll create one possibly today or in the next couple of days.

**Comment by philipnbbc** [comment]
The [v1.3 release](https://github.com/bbc/bmx/releases/tag/v1.3) is now available.

**Comment by irieger** [comment]
Thanks. No worries for the delay.
